### PR TITLE
Demonstrate yaml file #2

### LINKED ISSUE #2: Demonstrate creating YAML file with explicit reference to schema.
Modify the `README` to include steps (with images) for creating YAML file (in the root of a cloned version of this repo) with the first line specifying which schema the file must match.

Any screenshots should go into an `images` sub-folder, and have useful titles that match the contents.

### PR DESCRIPTION
**Pull request by ghost**
fixes #2 
-Create new `schema.json` file from `mssile-strike.schema.json`
-Make example yaml data called `missile-strike.yaml` referencing `schema.json`

### CODE REVIEW

**Comment by IanMayo** [comment]
Hello @Talented0217 - did you intend to close this early?

File: missile-strike.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+supports: "Supports main operation by degrading enemy air defense."
+activityCard: "FR Cyper Op 12 - Green Drain"
+rulesOfEngagement: "Weapons Free"
+emconState: "Electronic Silence"
+requiresAirToAir: false
+ownAssets:
+  - asset: "ta"
+    missileType: "SRBM"
+    number: 2
+otherAssets:
```
Comment (IanMayo) [inline missile-strike.yaml:19]: This is good work, but could you create multiple entries to ownAssets and otherAssets , so show it´s an array?

File: schemas/schema.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+          "number": {
+            "type": "number",
+            "title": "Quantity"
+          }
+        },
+        "required": ["asset", "number"]
+      },
+      "title": "Targets"
+    },
+    "ownAssets": {
```
Comment (IanMayo) [inline schemas/schema.json:95]: thanks for tidying up / converting this @Talented0217  - but could you include the definition for ownAssets?

File: missile-strike.yaml (hunk 1 above)
Comment (ghost) [reply missile-strike.yaml:19]: Okay, No problem

File: schemas/schema.json (hunk 2 above)
Comment (ghost) [reply schemas/schema.json:95]: Sure

File: missile-strike.yaml (hunk 1 above)
Comment (ghost) [reply missile-strike.yaml:19]: I've done that

File: schemas/schema.json (hunk 2 above)
Comment (ghost) [reply schemas/schema.json:95]: I've fixed it

**Pull request closed without merging**